_targets_eval_postprocessing.R

```diff
@@ -435,7 +435,7 @@ head_to_head_targets <- list(
         epidemic_phases,
         by = c(
           "location" = "state_abbr",
-          "date" = "reference_date"
+          "forecast_date" = "reference_date"
```

Comment:
question: why is the relevant date column of the `epidemic_phases` dataframe called `reference_date`? Is that what it's called in the source file or do we rename it that somewhere?

Reply:
Ah yeah this is from the retrospective R(t) and I just kept NNH's nomenclature (they use reference date and report date). I can change it when `epidemic_phases` gets made though

Reply:
Fixed this